src/io/sb3/fromSb3.ts

```diff
@@ -161,9 +161,10 @@ function getBlockScript(blocks: { [key: string]: sb3.Block }) {
               const shadowInputs = translateInputs(blocks[value].inputs);
               const shadowFields = translateFields(blocks[value].fields, blocks[value].opcode);
 
-              if (blocks[value].opcode === "pen_menu_colorParam") {
+              if (blocks[value].opcode === OpCode.pen_menu_colorParam) {
                 result = {
                   ...result,
+                  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
                   COLOR_PARAM: (shadowFields as any).colorParam
```

Comment:
Seems evil, obviously, but I'm OK with not addressing existing logic/safety issues as part of this PR. This instance sounds related to #100. Is it surprising or expected, that this is basically the only related line that new TypeScript checks caught?

Reply:
I believe so, yes.